argmin/src/solver/quasinewton/lbfgs.rs

```
@@ -138,6 +148,130 @@ where
         self.tol_cost = tol_cost;
         Ok(self)
     }
+
+    /// Sets coefficient of L1-regularization.
+    ///
+    /// `0.0` or negative value disables L1-regularization. Defaults to `0.0`.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use argmin::solver::quasinewton::LBFGS;
+    /// # use argmin::core::Error;
+    /// # fn main() -> Result<(), Error> {
+    /// # let linesearch = ();
+    /// let lbfgs: LBFGS<_, Vec<f64>, Vec<f64>, f64> = LBFGS::new(linesearch, 3).with_l1_regularization(1.0);
+    /// # Ok(())
+    /// # }
+    pub fn with_l1_regularization(mut self, l1_coeff: F) -> Self {
```

Please return an error if `l1_coeff <= 0.0`.